src/ucs/sys/event_set.c

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (C) Hiroyuki Sato. 2019. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include "event_set.h"
+
+#include <ucs/debug/memtrack.h>
+#include <ucs/debug/log.h>
+#include <ucs/debug/assert.h>
+
+#include <sys/epoll.h>
+#include <string.h>
+#include <errno.h>
+#include <unistd.h>
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#define UCS_EVENT_EPOLL_MAX_EVENTS 16
```

Comment:
@yosefe Why it is 16 ?

Reply:
@shamisp JFYI This value just copied from https://github.com/openucx/ucx/blob/master/src/ucs/async/thread.c#L20